### ISSUE TITLE
Possible to execute on all gpus in parallel?

### ISSUE THREAD

**Issue by fire** [nod-ai/SHARK#1180]
Can you enhance the demo to allow execution of all the gpus in parallel?

**Comment by powderluv** [comment]
yes we have some examples of that for LLMs. What model do you want to do that with ?
